glean-core/ios/Glean/Glean.swift

```diff
@@ -115,8 +116,12 @@ public class Glean {
                 return
             }
 
-            if self.debugViewTag != nil {
-                _ = self.setDebugViewTag(self.debugViewTag!)
+            if let debugViewTag = self.debugViewTag {
+                self.setDebugViewTag(debugViewTag)
```

Comment:
Since this _does_ return a value that we are ignoring, you probably need to leave the `_ =` to satisfy the compiler/lint warning.

Reply:
But it looks like it passed so nevermind...  /shrug